build.gradle

```diff
@@ -1,72 +1,76 @@
-buildscript {
-  repositories {
-    jcenter()
-  }
-}
-
 plugins {
     id 'java'
     id 'application'
     id 'base'
     id 'nebula.lint' version '9.3.4'
 }
-
-allprojects {
-  apply plugin: 'java'
-  apply plugin: 'base'
-  apply plugin: 'nebula.lint'
-  
-  repositories {
-    jcenter()
-  }
-
-  group = 'org.apereo.openequella.adminconsole'
-  version = artifactVersion
-
-  gradleLint.rules = ['all-dependency', 'dependency-parentheses', 'duplicate-dependency-class']
-
-  dependencies {
-    implementation 'org.slf4j:slf4j-api:1.7.26'
-    runtime 'org.slf4j:slf4j-simple:1.6.4'
-  }
-}
-
+/*
 subprojects {
   task sourcesJar(type: Jar) {
         from sourceSets.main.java
         archiveClassifier = 'sources'
     }
-}
+} */
 
-project(':common') {
-  dependencies {
-      implementation 'com.google.guava:guava:27.0.1-jre'
-      implementation 'com.fasterxml.jackson.core:jackson-databind:2.9.8'
-      implementation 'org.jasypt:jasypt:1.9.2'
-  }
+repositories {
+  jcenter()
 }
```

Comment:
probably be good to still have things like `mavenLocal()` so we don't always have to go remote - or is the diff confusing me here.